net/pdnsd/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
I do not think a copy of GPL version 3 file is necessary

Reply:
ok, i remove it.

Reply:
From a security point of view, PDNSD is to prevent DNS poisoning and cache to speed up the correct DNS resolution locally, so use query_method = tcp_only by default out of the box, otherwise there is no need to install PDNSD, which is why PDNSD exists.